.gitignore

```diff
@@ -0,0 +1,6 @@
+
```

Comment:
nit: I would strip out the blank lines here.

Reply:
Thanks, and removed.